### DISCUSSION TITLE
How much disk space is released when I would delete backup xyz?

### DISCUSSION THREAD

**Discussion by christfried** [borgbase/vorta#1283] [FAQ] [answered]
Doesn't the "size" column only show what got backed up that time? What if I delete some files from that exact backup state afterwards, back up again, and then delete this backup. The size shown would not be the disk space released anymore, am I right?
Where lies the benefit of having this column? I would much rather know how much disk space would get released following the deletion.

**Comment by m3nu** [comment, marked as answer]
Not possible to tell without very expensive calculations. There is also no disk space released, until you do `borg compact`.

**Reply by christfried** [reply]
> There is also no disk space released, until you do borg compact.

How can I do that? I would expect that disk space gets released whenever I am deleting an archive ...

**Reply by real-yfprojects** [reply]
This is the way borg handles it (for perfomance reasons?). You can run `compact` from the *archive tab* in vorta.

**Comment by real-yfprojects** [comment]
Related to #497.
